src/components/Card.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import '../index.css';
```

Comment:
You should import index.css in index.js.
It will apply the rules of a stylesheet on every descendant component.